### PR TITLE
VideoTile label overflow text ellipses

### PR DESCRIPTION
**Pull request by mgamis-msft**
# What

VideoTile label overflow text ellipses when text exceeds available space.

# Why

https://skype.visualstudio.com/SPOOL/_workitems/edit/2629760

# How Tested

Storybook testing:
https://user-images.githubusercontent.com/79475487/141005853-c5cd1cbf-cde5-4df9-b7eb-b8dc985afa94.mp4

Calling sample testing for mobile to cover case in bug ticket. 
https://user-images.githubusercontent.com/79475487/141006028-f7932060-346d-4aa7-88ff-2295c7cc2720.mp4
No label is shown on VideoTile as designed in Figma
image

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/src/components/VideoTile.tsx [hunk 1]
```diff
@@ -13,9 +13,9 @@ import {
   iconContainerStyle,
   overlayContainerStyles,
   rootStyles,
-  tileInfoStackItemStyle,
```
Comment (JamesBurnside) [inline packages/react-components/src/components/VideoTile.tsx:16]: Can this style be deleted now? Or is it used elswhere?

File: packages/react-components/src/components/styles/VideoTile.styles.ts [hunk 2]
```diff
@@ -37,53 +37,66 @@ export const overlayContainerStyles: IStyle = {
...
 };
 
+/**
+ * @private
+ */
+export const tileInfoContainerStyle = mergeStyles({
+  position: 'absolute',
+  bottom: '0.5rem',
+  left: '0.5rem',
+  minWidth: 'calc(100% - 1rem)',
```
Comment (JamesBurnside) [inline packages/react-components/src/components/styles/VideoTile.styles.ts:47]: is setting minWidth and maxWidth to the same value different than setting just width?

**Review by JamesBurnside — APPROVED** [review]

**Comment by prprabhu-ms** [comment]
Is the larger overlay size intended?

image

**Comment by prprabhu-ms** [comment]
> Is the larger overlay size intended?
> 
> image

Err -> smaller (it's larger before this PR)

File: packages/react-components/src/components/VideoTile.tsx (hunk 1 above)
Comment (mgamis-msft) [reply packages/react-components/src/components/VideoTile.tsx:16]: > > Is the larger overlay size intended?
> > image
> 
> Err -> smaller (it's larger before this PR)

Will try to maintain the old padding. Thanks.

File: packages/react-components/src/components/styles/VideoTile.styles.ts (hunk 2 above)
Comment (mgamis-msft) [reply packages/react-components/src/components/styles/VideoTile.styles.ts:47]: I think setting minWidth and maxWidth is much more strict for the children inside. Let me verify.

File: packages/react-components/src/components/VideoTile.tsx (hunk 1 above)
Comment (mgamis-msft) [reply packages/react-components/src/components/VideoTile.tsx:16]: Yes . This was used for Stack.Item that I removed with style {display: 'flex'} which didn't really do anything.

**Comment by mgamis-msft** [comment]
> > Is the larger overlay size intended?
> > image
> 
> Err -> smaller (it's larger before this PR)

Will try to maintain the old padding. Thanks.

**Review by anjulgarg — APPROVED** [review]

**Pull request merged**